Code/GraphMol/FMCS/FMCS.cpp

```diff
@@ -87,13 +91,25 @@ MCSResult findMCS_P(const std::vector<ROMOL_SPTR>& mols,
   return findMCS(mols, &p);
 }
 
+MCSResult findMCS(const std::vector<ROMOL_SPTR>& mols, bool maximizeBonds,
+        double threshold, unsigned timeout, bool verbose,
+        bool matchValences, bool ringMatchesRingOnly,
+        bool completeRingsOnly, bool matchChiralTag,
+        AtomComparator atomComp, BondComparator bondComp) {
+  return findMCS(mols, maximizeBonds, threshold, timeout, verbose,
+          matchValences, ringMatchesRingOnly, completeRingsOnly,
+          matchChiralTag, atomComp, bondComp, IgnoreRingFusion);
+}
+
 MCSResult findMCS(const std::vector<ROMOL_SPTR>& mols, bool maximizeBonds,
                   double threshold, unsigned timeout, bool verbose,
                   bool matchValences, bool ringMatchesRingOnly,
                   bool completeRingsOnly, bool matchChiralTag,
-                  AtomComparator atomComp, BondComparator bondComp) {
+                  AtomComparator atomComp, BondComparator bondComp,
+                  RingComparator ringComp) {
   // AtomComparator atomComp=AtomCompareElements;
```

Comment:
these commented out definitions could be removed

Reply:
yeah, I know they were in the original code. I think it's fine to remove them, but they don't need to go.